pkg/agent/cli/agent.go

```diff
@@ -15,19 +15,19 @@ import (
 )
 
 type Agent struct {
-	cfg            *config.Config
```

Comment:
@petethepig 
Do we want to keep the config.Config struct?
If my understanding is correct, the lifetime of all the config structs are withing the `Start(cfg *config.Config)` function in cli.go, so maybe we don't need an extra container for all the configs?

Reply:
Yeah, good point, we might do that in the future